Copy improvements (#443)
opentensor/developer-docs: docs/evm-tutorials/index.md

import { FaMoneyBillTransfer } from "react-icons/fa6";
import { GrStakeholder } from "react-icons/gr";

# EVM smart contracts on Bittensor

Full Ethereum virtual machine (EVM) compatibility is now available on subtensor (the blockchain in Bittensor). This allows users to:

> - Deploy any Ethereum smart contract on subtensor without changing the code

Reviewer: ```suggestion
- Deploy most EVM smart contracts on subtensor without changing the code
```
Maybe not necessary, but we don't necessarily have all the features the Ethereum does. @gztensor would know specifics. But we can maybe caveat it, or not.

> - Interact with deployed smart contract on the subtensor blockchain

Reviewer: ```suggestion
- Interact with deployed smart contracts on the subtensor blockchain
```
